Makefile

```diff
@@ -0,0 +1,19 @@
+VERSION=$(shell cat emscripten-version.txt | sed s/\"//g)
+DISTDIR=../emscripten-$(VERSION)
+EXCLUDES=tests site __pycache__ node_modules docs Makefile
```

Comment:
maybe it's better to have an explicit list of what is needed? I'm fine either way though.

Reply:
Btw, I searched for usage of `tests/` outside of the test suite, and found this in emcc.py:
```python
      input_file = 'hello_world.c'
      cmd = [shared.PYTHON, sys.argv[0], shared.path_from_root('tests', input_file), '-v', '-c', '-o', temp_target] + args
```
in the code for `--cflags`. Looks like that's the only thing though.